openstack/compute/v2/flavors/requests.go

```diff
@@ -21,22 +21,12 @@ type ListOptsBuilder interface {
 //
 // If no IsPublic argument is supplied in the request, OpenStack treats the FlavorType as Project.
 type FlavorType string
+
 const (
-        Project FlavorType =  "true"
-        Private FlavorType = "false"
-        All FlavorType = "None"
+	Project FlavorType = "true"
```

Comment:
Oh yes, I did misunderstand you.  Are you suggesting that the constant named `Project` should instead be named, for example, `FlavorAccess`?  That didn't make sense to me, which is why I interpreted you the way I did.

I'm having a hard time thinking of more accurate and descriptive constant names than what I have.  That's a comment about my limitations rather than the virtues of the names I suggested.  I wish the  OS community had stuck with the `Tenant` vernacular.  I think `flavors.Tenant` is more immediately understood than `flavors.Project`.

Let me know how you would like me to proceed (including if it's "Margaret: propose something better".)